src/Loudness.js

```diff
@@ -0,0 +1,71 @@
+import Sound from "./Sound.js";
+
+const IGNORABLE_ERROR = ["NotAllowedError", "NotFoundError"];
+
+// https://github.com/LLK/scratch-audio/blob/develop/src/Loudness.js
+export default class LoudnessHandler {
+  constructor() {
+    // TODO: use a TypeScript enum
+    this.connectionState = "NOT_CONNECTED";
+  }
+
+  get audioContext() {
+    return Sound.audioContext;
+  }
+
+  async connect() {
+    // If we're in the middle of connecting, or failed to connect,
+    // don't attempt to connect again
+    if (this.connectionState !== "NOT_CONNECTED") return;
+    this.connectionState = "CONNECTING";
+
+    try {
+      const stream = await navigator.mediaDevices.getUserMedia({ audio: true });
+      this.hasConnected = true;
+      this.audioStream = stream;
+      const mic = this.audioContext.createMediaStreamSource(stream);
+      this.analyser = this.audioContext.createAnalyser();
+      mic.connect(this.analyser);
```

Comment:
```suggestion
      mic.connect(this.analyser);
      
```

Reply:
Fixed that. I'd definitely like to revisit this again in the future, possibly by making some sort of audio counterpart to the `Renderer` that can be instantiated rather than relying on a singleton `AudioContext`.